<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketNotificationWidget</class>
 <widget class="QWidget" name="RocketNotificationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-family: &quot;Arial&quot;;
	font-size: 19px;
	color: rgb(50,50,50);
}

QCheckBox, QPushButton {
	font-family: &quot;Arial&quot;;
	font-size: 16px;
	color: rgb(80,80,80);
}

QCheckBox:hover {
	color: rgb(50,50,50);
}

QCheckBox::indicator:unchecked {
	image: url(:/images/icon-unchecked.png);
 }

QCheckBox::indicator:checked {
	image: url(:/images/icon-checked.png);
 }

QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(232, 234, 233, 255), stop:1 rgba(225, 226, 227, 255));
	color: rgb(70,70,70);
	border: 1px solid grey;
	border-radius: 3px;
	min-width: 80px;
	min-height: 30px;
	padding-left: 8px;
	padding-right: 8px;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(242, 244, 243, 255), stop:1 rgba(225, 226, 227, 255));
	color: rgb(60,60,60);
	border: 1px solid grey;
}

QPushButton:pressed {
	background-color: rgba(200, 200, 200);
	border: 1px solid grey;
}

QPushButton#buttonClose {
	background-image: url(:/images/icon-x-hover-24x24.png);
	background-position: center center;
	background-repeat: no-repeat;
	background-color: transparent;
	border: 0px;
	max-width: 32px;
	min-width: 32px;
	padding: 0px;
}

QPushButton#buttonClose:hover {
	background-image: url(:/images/icon-x-24x24.png);
}

QPushButton#buttonClose:pressed {
	background-image: url(:/images/icon-x-hover-24x24.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="labelIcon">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="buttonClose">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RocketPlugin.qrc"/>
 </resources>
 <connections/>
</ui>
